deployments/admission-controllers/scheduler/admission_util.sh

```diff
@@ -46,6 +46,42 @@ delete_resources() {
   return 0
 }
 
+updateEnvVars() {
+  # update environment variables
+  while true; do
+    case $1 in
+      -e|--env)
+        shift
+        if [ -z "$1" ]; then
+          echo "empty argument to -e|--env flag"
+          exit 1
+        fi
+        setEnvCmd=$1
+        case $setEnvCmd in
+          *=*)
+            echo "Update environment variable: $setEnvCmd"
+            eval "$setEnvCmd"
```

Comment:
why not export the environment variables here? 
so we don't need that in `create_resources()`

Reply:
I have tried to export the environment variables in preStart/exec/command but it seems not supported since scheduler pod failed and shown file "export xxxx=..." not exist. Also can't find any places to set the environments.